features/checkout/registering_account_after_checkout.feature

```diff
@@ -20,5 +21,19 @@ Feature: Registering a new account after checkout
         And I specify a password as "sylius"
         And I confirm this password
         And I register this account
-        And I verify my account using link sent to "john@example.com"
+        Then I should be on registration thank you page
+        When I verify my account using link sent to "john@example.com"
         Then I should be able to log in as "john@example.com" with "sylius" password
```

Comment:
We shouldn't mix `Then`, `When` and `Then` steps in a single scenario. Some steps should be changed or the logic should be split into two scenarios